swap/swap.go

```diff
@@ -491,6 +491,30 @@ func (s *Swap) ReceivedCheques() (map[enode.ID]*Cheque, error) {
 	return cheques, nil
 }
 
+// LiquidBalance returns the liquidBalance of the chequebook of the peer. If no peer is given, it will return the balance of the node itself
+func (s *Swap) LiquidBalance(peer enode.ID) (int64, error) {
+	var err error
+	if peer == (enode.ID{}) {
+		balance, err := s.contract.LiquidBalance(nil)
+		if err != nil {
+			return 0, err
+		}
+		return balance.Int64(), nil
+	}
+	var otherSwap contract.Contract
+	if swapPeer := s.getPeer(peer); swapPeer != nil {
+		otherSwap, err = contract.InstanceAt(swapPeer.contractAddress, s.backend)
+		if err != nil {
+			return 0, err
+		}
+	}
+	balance, err := otherSwap.LiquidBalance(nil)
+	if err != nil {
```

Comment:
this block of code appears 3 times in 20 lines. you could call it just once, like this:
```
if peer === (enode.ID{}){
    ...
}else{
    ...
}
...
if err != nil{
    return 0, err
}
```